arbitrum-docs/partials/_migrate-nft-collections-partial.md

```diff
@@ -0,0 +1,90 @@
+This how-to is for **users** and **developers** who want to migrate (or "bridge") existing NFT collections onto Arbitrum.
```

Comment:
@amarrazza I've explicitly indicated the target audience as being for both users _and_ developers here. We could create two docs - one for each audience - but that seems like an unnecessary optimization at this point.

Reply:
This documentation isn't really relevant to end users. For a user, there isn't a good reason to bridge an NFT to another chain unless the project migrates there. This is because all the utility and liquidity will be on the collection's canonical chain. From my perspective, this how-to is most useful for a developer or team member interested in migrating an NFT collection